app/src/main/java/com/zeapo/pwdstore/autofill/oreo/ui/InlineResponseBuilder.kt

```diff
@@ -0,0 +1,10 @@
+package com.zeapo.pwdstore.autofill.oreo.ui
+
+import android.app.PendingIntent
+import android.graphics.drawable.Icon
+import android.os.Build
+import android.service.autofill.InlinePresentation
+import android.widget.inline.InlinePresentationSpec
+import androidx.autofill.inline.UiVersions
+import androidx.autofill.inline.v1.InlineSuggestionUi
+
```

Comment:
I guess this file shouldn't have been here

Reply:
Right, deleted it.